src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
```

Comment:
Put all elements in this file like nested elements but not deeper than 3 layers, better not to use & because this rule could change some elements if more elements would be on a site(with __photo in name). 
Hint to discover: https://mate-academy.github.io/style-guides/htmlcss.html#nested-selectors